### PR TITLE
Generate django project files, basic django helloworld page up

### PR DESCRIPTION
**Pull request by joeboxberger21**
Based on requirements for sprint 0, the basic welcome page for django appears when we request the server. Generated all the basic project files for setup.

### CODE REVIEW

**Comment by joeboxberger21** [comment]
Fixed github actions issues

File: .github/workflows/django.yml [hunk 1]
```diff
@@ -29,8 +29,8 @@ jobs:
 
     - name: Style Checking
       run: |
-        flake8 . --max-line-length=79  # Customize max line length as needed
+        flake8 . --ignore=F401 --per-file-ignores="./port_inspector/port_inspector/settings.py:E501" --max-line-length=79
```
Comment (AudreyKrishnadasan) [inline .github/workflows/django.yml:32]: good switch on ignoring the specific error code rather than all error codes

**Review by AudreyKrishnadasan — COMMENTED** [review]
looks great - good workaround on the flake ignore stuff!

**Review by AudreyKrishnadasan — APPROVED** [review]
forgot to approve on the last review!

**Pull request merged**